docs/guides/jupyter.rst

```diff
@@ -0,0 +1,377 @@
+  documentation <https://nbsphinx.readthedocs.io/en/latest/a-normal-rst-file.html#thumbnail-galleries>`_.
+- From a Jupyter notebook, adding a ``"nbsphinx-gallery"`` tag to the metadata of a cell.
+  Each editor has a different way of modifying the cell metadata (see figure below).
+
+.. figure:: /_static/images/guides/jupyterlab-metadata.png
+   :width: 80%
+   :align: center
+   :alt: Panel to modify cell metadata in JupyterLab
+
+   Panel to modify cell metadata in JupyterLab
+
+For example, this reST markup would create a thumbnail gallery
+with generic images as thumbnails,
+thanks to the Sphinx-Gallery default style:
+
+.. code-block:: rest
+
+   Thumbnails gallery
+   ==================
+
+   .. nbgallery::
+      notebooks/Example 1
+      notebooks/Example 2
+
+.. figure:: /_static/images/guides/thumbnail-gallery.png
+   :width: 80%
+   :align: center
+   :alt: Simple thumbnail gallery created using nbsphinx
+
+   Simple thumbnail gallery created using nbsphinx
+
+To see some examples of notebook galleries in the wild:
+
+- `poliastro <https://docs.poliastro.space/>`_ offers tools for interactive Astrodynamics in Python,
+  and features `several examples and how-to guides using
+  notebooks <https://docs.poliastro.space/en/stable/gallery.html>`_
+  and displays them in an appealing thumbnail gallery.
+  In addition, `poliastro uses unpaired MyST
+  Notebooks <https://github.com/poliastro/poliastro/tree/0.15.x/docs/source/examples>`_
+  to reduce repository size and improve integration with git.
+
+Background on alternative notebook formats
+------------------------------------------
+
+Jupyter notebooks in ``.ipynb`` format
+(as described in the `nbformat
+documentation <https://nbformat.readthedocs.io/en/latest/>`_)
+are by far the most widely used for historical reasons.
+
+However, to compensate some of the disadvantages of the ``.ipynb`` format
+(like cumbersome integration with version control systems),
+`jupytext`_ offers `other formats <https://jupytext.readthedocs.io/en/latest/formats.html>`_
+based on plain text rather than JSON.
+
+As a result, there are three modes of operation:
+
+- Using classic ``.ipynb`` notebooks. It's the most straightforward option,
+  since all the tooling is prepared to work with them,
+  and does not require additional pieces of software.
+  It is therefore simpler to manage, since there are fewer moving parts.
+  However, it requires some care when working with Version Control Systems (like git),
+  by doing one of these things:
+
+  - Clear outputs before commit.
+    Minimizes conflicts, but might defeat the purpose of notebooks themselves,
+    since the computation results are not stored.
+  - Use tools like `nbdime <https://nbdime.readthedocs.io/>`_ (open source)
+    or `ReviewNB <https://www.reviewnb.com/>`_ (proprietary)
+    to improve the review process.
+  - Use a different collaboration workflow that doesn't involve notebooks.
+
+- Replace ``.ipynb`` notebooks with `a text-based
```

Comment:
One think that maybe be worth noting here, is that if you use markdown files in myst-nb, error/warning messages will point to the correct line in the source file: https://myst-nb.readthedocs.io/en/latest/use/start.html#sphinx-error-reporting

Reply:
Thanks!